tasks/main.yml

```diff
@@ -44,6 +44,16 @@
   notify:
     - Restart sftd
 
+- name: Override sftd.service
+  ansible.builtin.copy:
+    dest: /etc/systemd/system/sftd.service.d/override.conf
+    content: |
+      [Service]
+      OOMScoreAdjust=-1000
+    mode: 0644
+  notify:
+    - Restart sftd
```

Comment:
We should also add `daemon_reload: true` to the restart handler for good measure, see https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html#parameter-daemon_reload